test/configure-options.test.ts

```diff
@@ -78,7 +78,11 @@ function traverseCO(root: any, rootPath: string[] = []): void {
                 typeof v === "boolean"
             )
           );
-        } else if (typeof value === "boolean" || typeof value === "string") {
+        } else if (
+          typeof value === "boolean" ||
+          typeof value === "string" ||
+          typeof value === "number"
+        ) {
```

Comment:
What does number mean?

Reply:
It was to handle the non-range min/max values. I've reverted this change.

Reply:
Just to explain: This wouldn't handle anything. The purpose of this test is to catch some common mistakes (like misunderstaing the four number tuple syntax). It would still fail in runtime, you just wouldn't be warned in advance.